setup.py

```diff
@@ -75,10 +74,11 @@
     # pip to create the appropriate form of executable for the target platform.
     entry_points='''
         [ckan.plugins]
-        data_qld_resources=ckanext.data_qld.resources_plugin:DataQldResourcesPlugin
-        data_qld_integration=ckanext.data_qld.integration_plugin:DataQldIntegrationPlugin
+        data_qld=ckanext.data_qld.plugin:DataQldPlugin
+        data_qld_integration=ckanext.data_qld.plugin:DataQldPlugin
```

Comment:
is it ok to load our plugin multiple times?

Reply:
Not sure. I'm testing that now.

Reply:
Looks like CKAN doesn't mind.